gamatrix-gog.py

```diff
@@ -369,91 +380,46 @@ def set_multiplayer_status(game_list, cache):
             and "game_modes" in cache["igdb"]["games"][igdb_key]["info"][0]
         ):
             if cache["igdb"]["games"][igdb_key]["info"][0]["game_modes"] == [
-                IGDB_GAME_MODE["singleplayer"]
+                constants.IGDB_GAME_MODE["singleplayer"]
             ]:
                 max_players = 1
                 reason = "as IGDB has single player as the only game mode"
             else:
                 for mode in cache["igdb"]["games"][igdb_key]["info"][0]["game_modes"]:
-                    if mode in IGDB_MULTIPLAYER_GAME_MODES:
+                    if mode in constants.IGDB_MULTIPLAYER_GAME_MODES:
                         multiplayer = True
                         reason = f"as game modes includes {mode}"
                         break
 
         log.debug(
-            "{} ({}, IGDB key {}): multiplayer {}, max players {} {}".format(
-                k, game_list[k]["title"], igdb_key, multiplayer, max_players, reason
-            )
+            f"{k} ( {game_list[k]['title']}, IGDB key {igdb_key}): ",
```

Comment:
Now you're gonna see what a pedant I am: there's a space after the `(` that will make my eyelid twitch :-)